packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -541,12 +565,36 @@ export class StatsAnalyzer extends EventsScope {
               `StatsAnalyzer:index#compareLastStatsResult --> No audio Energy present`,
               currentStats.totalAudioEnergy
             );
+            if (this.lastEmittedStartStopEvent.audio.remote === EVENTS.LOCAL_MEDIA_STARTED) {
```

Comment:
same like my other comment - `currentStats` is set to `this.statsResults['audio-send'].send` so it's about local stuff not remote

Reply:
Changed.